test/shared/includes.js

```diff
@@ -8,24 +8,38 @@ exports.test = function(name) {
 
   describe(name, function(){
 
-    it('should support includes', function(done) {
-      var str = fs.readFileSync('test/fixtures/' + name + '/include.' + name).toString();
+    if (name === 'liquid') {
+      it('should support includes', function (done) {
+        var str = fs.readFileSync('test/fixtures/' + name + '/include.' + name).toString();
 
-      var locals = { user: user, includeDir: 'test/fixtures/' + name };
+        var locals = { user: user, includeDir: 'test/fixtures/' + name };
 
-      cons[name].render(str, locals, function(err, html){
-        if (err) return done(err);
-        try{
-          if (name === 'liquid') {
```

Comment:
This could be generalized with a switch case for the name and depending on the name of the lib it'll return different values the html should return. Or the default value.